js/src/predicate.ts

```diff
@@ -256,6 +270,8 @@ export class CustomPredicate extends Predicate {
 
 export function lit(v: any): Value<any> { return new Literal(v); }
 export function col(n: string): Col<any> { return new Col(n); }
+export function and(...p: Predicate[]): Predicate { return new And(...p); }
+export function or(...p: Predicate[]): Predicate { return new Or(...p); }
```

Comment:
Should these return types also be `And` and `Or` respectively, to match the definitions above?